docs/events.md

```diff
@@ -1,20 +1,44 @@
-## Local Testing of Cloud Events
+# Events
 
-The setup for cloud functions that accept events is very similar to the instructions in the quickstart, with the following adjustments:
+This guide shows you how to use the Functions Framework for local testing with:
 
-In your `package.json`, add a signature type (in bold) to your start command:
+- CloudEvents
+- The Pub/Sub Emulator
 
-<pre>
+## Local Testing of CloudEvents
+
+In your `package.json`, specify `--signature-type=event"` for the `functions-framework`:
+
+```sh
+{
   "scripts": {
-    "start": "functions-framework --target=helloWorld <b>--signature-type=event"</b>
+    "start": "functions-framework --target=helloWorld --signature-type=event"
   }
-</pre>
+}
+```
 
-Upon running ```sh npm start ```, you'll see the function is still being served at `http://localhost:8080/`. However, it is no longer accessible via GET requests from the browser. Instead, send a POST request where the request body conforms to the API defined by [push subscriptions](https://cloud.google.com/pubsub/docs/push). 
+Create an `index.js` file:
 
-### Submitting POST Request to Simulate Pub/Sub messages
+```js
+exports.helloPubSub = (data, context) => {
+  console.log(data);
+}
+```
+
+Start the Functions Framework:
+
+```sh
+npm start
+```
 
-Create a `mockPubsub.json` file with the following contents:
+Your function will be serving at `http://localhost:8080/`, however,
+it is no longer accessible via `HTTP GET` requests from the browser.
```

Comment:
What happens if you send it a GET request?

Reply:
You get a standard HTTP 404 response with Express' default `Cannot GET ${PATH}` error. I don't think this is worth additionally documenting/reading.